### ISSUE TITLE
Remove readyToReview

### ISSUE THREAD

**Issue by LegoStormtroopr** [aristotle-mdr/aristotle-metadata-registry#624]
There are a few reference to deprecated workflows that need to be removed.

```
legostormtroopr:~/workspace (master) $ grep readyToReview * -r
aristotle_mdr/utils/utils.py:    excluded_fields='_concept_ptr readyToReview version workgroup pk id supersedes superseded_by _is_public _is_locked'.split()
aristotle_mdr/templates/aristotle_mdr/actions/mark_ready_to_review.html:        {% if item.readyToReview %}
aristotle_mdr/templates/aristotle_mdr/actions/mark_ready_to_review.html:        {% if item.readyToReview %}
aristotle_mdr/templates/aristotle_mdr/helpers/paginated_list.html:            {% elif item.readyToReview %}
aristotle_mdr/tests/main/test_html_pages.py:            self.item1.readyToReview = True
```

**Comment by alodavi** [comment]
Hi! I would be glad to help :)

**Comment by LegoStormtroopr** [comment]
Hi @alodavi that'd be great.

This should be a really straightforward issue as this is just some dead code that isn't being used. Most of it is already in the issue, and  PR shouldn't show any test or coverage differences.

If you could clean this up I'd be very grateful!

**Comment by alodavi** [comment]
Alright. So if I understood it right mark_ready_to_review.html could be deleted completely? 
Moreover I saw that deleting `self.item1.readyToReview = True ` could affect some other code in `aristotle_mdr/tests/main/test_html_pages.py` . Is there a way to safely get rid of it in this situation?

**Comment by LegoStormtroopr** [comment]
Yup, the HTML file `mark_ready_to_review.html` needs to be removed, as well as the references in `test_html_pages.py`, `utils.py` and `paginated_list.html` - all of these are the files above and are quite small changes. They should be able to be removed safely, but when you check in your code and submit a pull request it will kick of a test build that can verify all of that for you.

For context, 'Ready To Review' was a flag on the `Concept` object before the entire review workflow process was added and has since been remove almost everywhere else.

**Comment by LegoStormtroopr** [comment]
Closed by #629
